tools/gulp/tasks/docs.ts

```diff
@@ -69,10 +69,25 @@ task('docs', [
 
 /** Generates html files from the markdown overviews and guides. */
 task('markdown-docs', () => {
+  // Extend the renderer for custom heading anchor rendering
+  markdown.marked.Renderer.prototype.heading = (text: string, level: number): string => {
```

Comment:
Because we are using the gulp plugin, I wasn't able to override the renderer like that. In order to do that, I would need to roll our own gulp adapter so we can pass in a custom renderer. Is that something you would like me to look at?